docs/package.json

```diff
@@ -0,0 +1,53 @@
+{
+  "name": "docs",
+  "version": "0.0.1",
+  "private": true,
+  "scripts": {
+    "dev": "docusaurus start --port 3200 --no-open",
+    "docusaurus": "docusaurus",
+    "start": "docusaurus start --port 3200",
+    "build": "docusaurus build",
+    "swizzle": "docusaurus swizzle",
+    "deploy": "docusaurus deploy",
+    "clear": "docusaurus clear",
+    "serve": "docusaurus serve",
+    "write-translations": "docusaurus write-translations",
+    "write-heading-ids": "docusaurus write-heading-ids",
+    "typecheck": "tsc"
+  },
+  "dependencies": {
+    "@docusaurus/core": "2.1.0",
+    "@docusaurus/preset-classic": "2.1.0",
+    "@docusaurus/theme-live-codeblock": "^2.1.0",
+    "@mdx-js/react": "^1.6.22",
+    "clsx": "^1.2.1",
+    "docusaurus-plugin-sass": "^0.2.2",
+    "modern-normalize": "^1.1.0",
+    "prism-react-renderer": "^1.3.5",
+    "react": "^18.2.0",
```

Comment:
I changed this from ^17.0.2 in order to get it to work in the docs/ folder. I'm not sure if 18 is officially supported by Docusaurus 2, and we might want to change it back.